app/services/induction/change_induction_record.rb

```diff
@@ -21,6 +21,7 @@ def call
 
         induction_record.changing!(time_now)
         new_record.assign_attributes(default_attrs.merge(changes))
+        new_record.start_date = new_record.end_date if new_record.start_date && new_record.end_date && new_record.start_date > new_record.end_date
```

Comment:
Do we risk setting induction records out of order if we are setting the previous induction record `end_date` to `induction_record.changing!(time_now)`, while the new induction record `end_date` and `start_date` might be in the past?

Reply:
There's quite a lot going on in this line too, worth setting a variable?